wordpress-mobile/WordPress-iOS
# Improve Cocoapods caching

Improves CocoaPods caching by:
- Using the more specialized CocoaPods-specific caching functions
- Setting up the infrastructure for automated cache rebuilds 

To test:
- Ensure CI passes
- Ensure that [the test scheduled task](https://buildkite.com/wordpress-mobile/wordpress-ios/builds/1681) passes as well

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a
PR submission checklist:
